airbyte-integrations/connectors/source-file/source_file/spec.json

```diff
@@ -51,6 +51,13 @@
     "dependencies": {
       "storage": {
         "oneOf": [
+          {
+            "properties": {
```

Comment:
How is this duplicating the REST source? It feels this source can handle what we do in the REST source, just a matter of adding some header options.

@cgardens what do you think?

Reply:
And this way we can handle the format we want (json/csv/...)

Reply:
those two feel slightly separate. I imagine in the evolution of the REST API source, we'll want to handle paging, backoff, rate limiting etc. different than how we'd handle a single file download. And from a marketing standpoint, I think that sounds confusing - this source already handles a very wide variety of things. Lumping in REST API as well would make it harder to market what the source does.